src/test/kotlin/com/forgerock/uk/openbanking/tests/functional/payment/domestic/vrp/api/v3_1_10/CreateDomesticVrp.kt

```diff
@@ -266,6 +268,22 @@ class CreateDomesticVrp(val version: OBVersion, val tppResource: CreateTppCallba
         assertThat((exception.cause as FuelError).response.responseMessage).isEqualTo(UNAUTHORIZED)
     }
 
+    fun shouldCreateDomesticVrpConsent_throwsBadRequestHasDifferentVrpTypeOtherThanSweepingTest() {
```

Comment:
Maybe the method name could be simplified.

Reply:
Made the requested changes.